Code/Tools/ProjectManager/Platform/Windows/ProjectBuilderWorker_windows.cpp

```diff
@@ -19,6 +19,7 @@ namespace O3DE::ProjectManager
         QString targetBuildPath = QDir(m_projectInfo.m_path).filePath(ProjectBuildPathPostfix);
 
         return AZ::Success(QStringList{ ProjectCMakeCommand,
+                                        "-G", "Visual Studio 16 2019",
```

Comment:
@amznestebanpapp @AMZN-Phil - should we leave this change out of dev so you can build with VS 2022 from the Project Manager?

Reply:
lets merge it so the history better reflects it (since this was done in RTE/stabilization). If we don't merge this in, then we will always get it as a change.

A different issue can remove this and the vswhere restriction. I think O3DE.exe should also let you choose a generator (in this case let you choose "Visual Studio 16" or "Visual Studio 17") and by default not pass any (in this case). From the command line is easy to choose, I think we should allow the user to choose from O3DE as well. Maybe that can be done with the "parameters change"?

Reply:
OK, sounds good, I'm OK with updating this as part of those upcoming features.